### ISSUE TITLE
plot_loss invoked without first training model

### ISSUE THREAD

**Issue by yaoyuan10475** [adityagoel4512/Spectral-normalized-Neural-Gaussian-Process-PyTorch#1]
`model = trainer.train(training_data=TrainDataset(X=train_examples, y=train_labels), data_loader_config=training_config, epochs=100)`
This sentence cannot be executed,do you know what the reason is?

**Comment by adityagoel4512** [comment]
My apologies for only just getting back to you on this @yaoyuan10475. I updated the model package to make the `train` function a generator with checkpoints. I'm updating `sngp.ipynb` to reflect this now.

**Comment by adityagoel4512** [comment]
93c60b1be46b4797f3a1fae846d2279ca1886faa on master branch is updated with `sngp.ipynb` showing example usage now.
